modules/op-creating-pipeline-templates-admin-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/working-with-pipelines-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="op-creating-pipeline-templates-admin-console_{context}"]
+= Creating pipeline templates in the Administrator perspective
+
+As a cluster administrator, you can create pipeline templates which developers can reuse when they create a pipeline on the cluster.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster with cluster administrator permissions, and have switched to the *Administrator* perspective.
+* You have installed the {pipelines-shortname} Operator in your cluster.
```

Comment:
Same here re tense

Reply:
See previous comment

Reply:
In plain simple prereqs removing "have" is usually the way to go, but this is minor, I will leave this to you